### PR TITLE
Set IsEnabled by a new property instead of setting it separately for …

### PR DESCRIPTION
**Pull request by qhy040404**
…each SettingsCard

### CODE REVIEW

**Comment by Lightczx** [comment]
@qhy040404 Have you compared which approach is faster and more memroy efficient?

**Comment by qhy040404** [comment]
26cfded263d2e4aa9cf11afffd2fcd5
0a29fbfd03700a379e82194b1cc35b6
The main memory and CPU overhead is found in reading LaunchOptions when first set up, other overhead is lower.

**Comment by Lightczx** [comment]
Then you should make this a Behavior<SettingsExpander> and do not directly use event handler in Page.xaml.cs
Sample codes can be found in https://github.com/DGP-Studio/Snap.Hutao/tree/main/src/Snap.Hutao/Snap.Hutao/Control/Behavior

**Comment by Lightczx** [comment]
My suggestion is:
having a AttachedProperty like `SettingsExpanderHelper.IsItemsEnabled`
bind this property in the master toggle
implement the logic in the `OnIsItemsEnabledChanged` eventhandler.
By doing so, we even don't need a Behavior

File: src/Snap.Hutao/Snap.Hutao/View/Page/LaunchGamePage.xaml [hunk 1]
```diff
@@ -202,25 +203,17 @@
                                 Description="{shcm:ResourceString Name=ViewPageLaunchGameArgumentsDescription}"
                                 Header="{shcm:ResourceString Name=ViewPageLaunchGameArgumentsHeader}"
                                 HeaderIcon="{shcm:FontIcon Glyph=&#xE943;}"
-                                IsExpanded="True">
+                                IsExpanded="True"
+                                shch:SettingsExpanderHelper.IsItemsEnabled="{Binding Options.IsEnabled}">
```
Comment (Lightczx) [inline src/Snap.Hutao/Snap.Hutao/View/Page/LaunchGamePage.xaml:207]: @qhy040404 where is `SettingsExpanderHelper`?

**Review by Lightczx — CHANGES_REQUESTED** [review]

File: src/Snap.Hutao/Snap.Hutao/View/Page/LaunchGamePage.xaml (hunk 1 above)
Comment (qhy040404) [reply src/Snap.Hutao/Snap.Hutao/View/Page/LaunchGamePage.xaml:207]: sorry for my carelessness

**Pull request merged**